doc/source/f2py/code/CMakeLists_skbuild.txt

```diff
@@ -16,74 +15,81 @@ if(PROJECT_SOURCE_DIR STREQUAL PROJECT_BINARY_DIR)
   )
 endif()
 
-# Grab Python
-find_package(Python3 3.9 REQUIRED
-  COMPONENTS Interpreter Development)
-
 # Ensure scikit-build modules
 if (NOT SKBUILD)
-  # Kanged -->https://github.com/Kitware/torch_liberator/blob/master/CMakeLists.txt
+  find_package(PythonInterp 3.7 REQUIRED)
+  # Kanged --> https://github.com/Kitware/torch_liberator/blob/master/CMakeLists.txt
   # If skbuild is not the driver; include its utilities in CMAKE_MODULE_PATH
   execute_process(
-  COMMAND "${Python3_EXECUTABLE}"
-  -c "import os, skbuild; print(os.path.dirname(skbuild.__file__))"
-  OUTPUT_VARIABLE SKBLD_DIR
-  OUTPUT_STRIP_TRAILING_WHITESPACE
+    COMMAND "${PYTHON_EXECUTABLE}"
+    -c "import os, skbuild; print(os.path.dirname(skbuild.__file__))"
+    OUTPUT_VARIABLE SKBLD_DIR
+    OUTPUT_STRIP_TRAILING_WHITESPACE
   )
-  set(SKBLD_CMAKE_DIR "${SKBLD_DIR}/resources/cmake")
-  list(APPEND CMAKE_MODULE_PATH ${SKBLD_CMAKE_DIR})
+  list(APPEND CMAKE_MODULE_PATH "${SKBLD_DIR}/resources/cmake")
+  message(STATUS "Looking in ${SKBLD_DIR}/resources/cmake for CMake modules")
 endif()
 
 # scikit-build style includes
 find_package(PythonExtensions REQUIRED) # for ${PYTHON_EXTENSION_MODULE_SUFFIX}
-find_package(NumPy REQUIRED) # for ${NumPy_INCLUDE_DIRS}
-find_package(F2PY REQUIRED) # for ${F2PY_INCLUDE_DIR}
```

Comment:
These packages are just a bit too broken to use for now. If they worked properly, the rest of this would be really simple.

Reply:
It was my understanding that they do run as is as well though o.O
Though they're not reliable in selecting the right versions.

Reply:
Well, if you select the wrong Python and it doesn't have a copy of NumPy (or an incorrect version of NumPy), then that's the equivalent of broken to me, since it doesn't work (for me, it was getting a different path than the one being used to build). Though yes, they do run.